site/migration/01-migration.md

```diff
@@ -0,0 +1,257 @@
+| kpt cfg cat DIR/STDIN [flags]                                                                    | Deprecated.                                                                                                                                                                                                                                      |
+| kpt fn run DIR/STDIN [flags]                                                                     | kpt fn eval [DIR / -] [flags] <br> Executes a single function on resources in a directory. <br> <br> kpt fn render [PKG_PATH] <br> Executes the pipeline of functions on resources in the package and writes the output to the local filesystem. |
+| kpt fn source DIR/ [flags]                                                                       | kpt fn source [DIR] [flags] <br> Reads resources from a local directory and writes them in Function Specification wire format to stdout.                                                                                                         |
+| kpt fn sink DIR/ [flags]                                                                         | kpt fn sink [DIR] [flags] <br> Reads resources from stdin and writes them to a local directory.                                                                                                                                                  |
+| kpt fn export DIR/ [flags]                                                                       | Deprecated                                                                                                                                                                                                                                       |
+| kpt live init DIR/ [flags]                                                                       | kpt live init [PKG_PATH] [flags] <br> Initializes the package with the name, namespace and id of the resource that will keep track of the package inventory.                                                                                     |
+| kpt live apply DIR/STDIN [flags]                                                                 | kpt live apply [PKG_PATH / -] [flags] <br> Creates, updates and deletes resources in the cluster to make the remote cluster resources match the local package configuration.                                                                     |
+| kpt live preview DIR/STDIN [flags]                                                               | kpt live preview [PKG_PATH / -] [flags] <br> Preview the changes being applied to the live cluster.                                                                                                                                              |
```

Comment:
preview and diff have been removed as well.

Reply:
Updated.